src/builders/polylines.js

```diff
@@ -40,579 +43,444 @@ export function buildPolylines (
         miter_limit
     }) {
 
-    var cornersOnCap = corners_for_cap[cap] || 0;         // default 'butt'
-    var trianglesOnJoin = triangles_for_join[join] || 0;  // default 'miter'
+    var cap_type = cap ? CAP_TYPE[cap.toUpperCase()] : CAP_TYPE.BUTT;
```

Comment:
It feels unnecessary here to have to do a string transform to uppercase here (for no functional difference as far as I can tell besides a naming convention). Can we just use the lowercase which is what the official style spec is, and avoid any extra work (related: it isn't actually specified that uppercase or mixed case cap/join values from the scene file should work, and I don't know what the ES behavior is).